lib/index.js

```diff
@@ -2,13 +2,73 @@
 
 // Load modules
 
-const Http = require('http');
 const Hoek = require('hoek');
 
 
 // Declare internals
 
-const internals = {};
+const internals = {
+    STATUS_CODES: {
+        __proto__: null,
```

Comment:
According to MDN yes, there's the same thing in examples : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/setPrototypeOf#Examples